### ISSUE TITLE
Error: array_key_exists(): The first argument should be either a string or an integer

### ISSUE THREAD

**Issue by Tinpro** [nWidart/laravel-modules#1015]
When I install the laravel module follow the instructions.
I reloaded the local page and got an error
array_key_exists(): The first argument should be either a string or an integer
how to fix it ?
Thank you very much...!
![image](https://user-images.githubusercontent.com/23215618/85991317-7a7c3600-ba1d-11ea-8ee6-46f06d3c5bd4.png)

**Comment by nWidart** [comment]
Hi,

I can't seem to reproduce this.
Can you see what is being sent to this argument to understand what's going wrong?

**Comment by manishgoyalchqbook** [comment]
Hi @Tinpro 
Were you able to get some solution for this?

**Comment by nWidart** [comment]
I'll close this as there hasn't been a reply in a while, feel free to comment if needed.

**Comment by veyo007** [comment]
What is a solution for issue above? Because I got the same error when running command: php artisan ser

**Comment by andersonpem** [comment]
I was just having this issue. The company I work for uses Laravel 5.7 as an application base. I tried to install laravel modules without specifying its version and this happened when I created a module.

At least in my case what fixed it was:

- Determining which version of Laravel is being used;

- Checking which laravel-modules version is suitable for it;

- Reinstalling laravel-modules and generate a new module or try to fix yours to fit, if it was built for a later version of Laravel.

Probably some change through the module's structure or Laravel itself breaks modules if you try to use an older version of the framework.
